runtime/src/bank/serde_snapshot.rs

```diff
@@ -394,10 +394,6 @@ mod tests {
         .unwrap();
 
         assert_eq!(bank.epoch_stakes, dbank.epoch_stakes);
-        assert_eq!(
-            bank.fee_rate_governor.lamports_per_signature,
-            dbank.fee_rate_governor.lamports_per_signature
-        );
```

Comment:
note to self: does removing `fee_rate_governor` as extra fields modified break the test integrity?